generated-dockerfiles/rapidsai-core_centos7-base.amd64.Dockerfile

```diff
@@ -58,6 +58,9 @@ WORKDIR ${RAPIDS_DIR}
 
 COPY NVIDIA_Deep_Learning_Container_License.pdf . 
 COPY entrypoint.sh /opt/docker/bin/entrypoint
+
+ENV "PATH=/opt/conda/envs/rapids/bin:/opt/conda/condabin:/opt/conda/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
We do something very similar in conda-forge. Though we do this in [the entrypoint script]( https://github.com/conda-forge/docker-images/blob/a4ecdc0d0e049dc121fa7f59ed97ec88c7238408/scripts/entrypoint#L14 )

Reply:
Yeah the challenge that we are trying to deal with is that many container platforms do not call the entrypoint script.

Reply:
Right had a similar discussion with @mmccarty offline

IIUC some call `docker run`, but may `docker exec` into the container as well. With `docker exec`, it is possible to prefix any command with the entrypoint script as part of the command, which would handle this case

Manipulating the `PATH` will largely work in most cases. Where it falls short is `conda activate` injects things into the current shell, which is lost without activation. Also any packages that have activation scripts won't be run. Usually this isn't a big deal, but sometimes it can cause issues

While I agree adding `PATH` was the right thing to do in the short-term, in the medium term we may want to improve on this further to ensure activation is used in these other cases. Solutions could range from having folks run the entrypoint and including that in docs to other fixes in the container that aid with activation. Not entirely sure all of the constraints, but maybe we can discuss those offline

Reply:
Sure I totally agree. I guess the issue is that the entrypoint is specifically a Docker thing, and there are many other container runtimes and platforms that folks are trying to use our images on (contained, cri-o, kubernetes, kubeflow, sagemaker, vertex, azureml, etc). Most do not call the entrypoint at all if a custom command is set, and many set custom commands that cannot be overridden. So we should try not to rely on the entrypoint as a way to correctly set up our environment.

I think this would mean we need to set things up in the base environment, or manually set the environment up to be in a "post conda activate` state when the container is started. Setting environment variables should be straight forwards, but I hadn't considered activation scripts. 

Let's chat in the team meeting today and we can feedback here or in #438/#439.